string id
float64[] timestamps
float64[] signal